types/three/examples/jsm/animation/CCDIKSolver.d.ts

```diff
@@ -3,16 +3,16 @@ import { Object3D, SkinnedMesh, Vector3 } from '../../../src/Three.js';
 // tslint:disable-next-line:interface-name
 export interface IKS {
     effector: number;
-    iteration: number;
+    iteration?: number;
```

Comment:
Should these be `?: number | undefined`?

Reply:
They can be. 😄

Reply:
I'd say probably if you are trying to make sure `exactOptionalPropertyTypes` works, but, up to you.

Reply:
Yeah, we don't follow the pattern consistently and haven't really gotten any complaints or feedback on it.

Reply:
Low usage of the option, I'm afraid.

Reply:
Indeed. I like the concept, but don't want to take on the burden of fixing all the types that don't conform to it.

Reply:
But I would support it if it were in `strict` by default.

Reply:
That's a chicken and egg problem; nobody uses it because it's annoying, it's annoying because the types aren't updated, so we can't make it the default, so people aren't willing to fix the types, and so on. Hence, I try my best to point it out, and we had previously updated all of DT to add `| undefined`.